### PR TITLE
Move to xenial on TravisCI, Drop Python 3.4, 3.5 and add Python 3.7 support

### LINKED ISSUE #14: Move travis to xenial, drop Django 1.8 and Python 3.5

### PR DESCRIPTION
**Pull request by EnTeQuAk**
This also adds Django 2.1 support.

Fixes #14 
Closes https://github.com/mozilla/django-post-request-task/pull/13

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,34 +1,27 @@
...
   exclude:
-    - python: '3.6'
-      env: DJANGO_VERSION=1.8.x
-    - python: '2.7'
-      env: DJANGO_VERSION=2.0.x
-    - python: '3.4'
+    - python: 2.7
       env: DJANGO_VERSION=2.0.x
-    - python: 'pypy'
+    - python: pypy
```
[the remark this answers is not in the document]
Comment (EnTeQuAk) [reply .travis.yml]: Whoops, sorry, I forgot about that. 6bed1f5

File: setup.py [hunk 2]
```diff
@@ -66,8 +66,8 @@ def read(*parts):
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.4',
-        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: Implementation :: PyPy',
```
[the remark this answers is not in the document]
Comment (EnTeQuAk) [reply setup.py]: Indeed we shouldn't, removed in 6bed1f5

**Comment by EnTeQuAk** [comment]
Completely forgot about this pull request, updated it to Django 2.2 and merged it accordingly.

@diox can I go ahead and release a package?

**Comment by diox** [comment]
Sure. Let me know if you don't have access.

**Pull request merged**